ip6defrag/defrag.go

```diff
@@ -0,0 +1,227 @@
+// Package ip6defrag implements a IPv6 defragmenter
+package main
+
+import (
+	"container/list"
+	"fmt"
+	"github.com/google/gopacket"
```

Comment:
Please separate standard-lib imports from gopacket imports, see https://github.com/google/gopacket/blob/master/ip4defrag/defrag.go for an example